src/content/docs/en/anywhere/surface.mdx

```diff
@@ -5,44 +5,55 @@ description: "Information about Ultramarine Linux on Microsoft Surfaces"
+| Surface Pro 4                | Yes                     |                                                                                                                                                    |               |
+| Surface Pro (5th Generation) | Yes                     | Nearly identical to Pro 4                                                                                                                          | ✓             |
+| Surface Pro 6                | Yes                     | Nearly identical to 5th gen                                                                                                                        |               |
+| Surface Pro 7                | Yes                     |                                                                                                                                                    |               |
+| Surface Pro 7+               | Yes                     |                                                                                                                                                    |               |
+| Surface Pro 8                | Yes                     |                                                                                                                                                    |               |
+| Surface Pro 9                | Yes                     |                                                                                                                                                    |               |
+| Surface 3                    | Yes                     | Battery and touch will not work without Surface Kernel                                                                                                           |               |
+| Surface Go                   | No                      | Cameras and LTE will not work without Surface Kernel                                                                                               | ✓             |
+| Surface Go 2                 | No                      | Same as Surface Go                                                                                                                                 | ✓             |
+| Surface Go 3                 | No                      |                                                                                                                                                    |               |
+| Surface Go 4                 | Yes                     | UFS Storage, requires some manual intervention to get working                                                                                      |               |
+| Surface Book                 | Yes                     | Also needs Nvidia drivers                                                                                                                          | ✓             |
```

Comment:
mention Nvidia drivers for for some SKUs, i'm just now finding out they sold a small number of SKUs of the Surface Books without Nvidia 

Also fans don't work